packages/backend/server/src/schema.gql

```diff
@@ -655,6 +719,9 @@ type Query {
   """Get workspace by id"""
   workspace(id: String!): WorkspaceType!
 
+  """Get workspace role permissions"""
+  workspaceRolePermissions(id: String!): WorkspaceRolePermissions!
```

Comment:
can we just put this in `WorkspaceType` ?

Reply:
merge the request in the frontend, it's the same